src/main/java/com/cmsc495/ticketsystem/service/EmailService.java

```diff
@@ -64,5 +63,6 @@ public void sendEmail(String toEmail, String subject, String body) throws Except
         } else {
             throw new Exception("Failed to refresh access token");
         }
+        return false;
```

Comment:
Not sure why we should return false here.

I do think changing this method to return a boolean can be better, but we want it to return true in the event the email is sent, and false in the event of an exception.

You're on the right track this is a good practice as it can be used in the controller for if statements.